### PR TITLE
fetch fix for IntSight

### PR DESCRIPTION
**Pull request by orenzohar**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17476

## Description
fetch process in IntSight was broken. now it works. added `last fetch` param.

## Does it break backward compatibility?
   - Yes
       - Further details:
   - No

## Must have
- [x] Tests
- [x] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

**Comment by orenzohar** [comment]
[intsight_changes.zip](https://github.com/demisto/content/files/3310997/intsight_changes.zip)
@yuvalbenshalom

File: Integrations/IntSight/IntSight.py [hunk 1]
```diff
@@ -64,7 +71,7 @@ def convert_python_date_to_unix_millisecond(python_date_object):
 
 
 def increase_iso_by_x_days(date_in_iso_format, num_of_days):
-    iso_format = "%Y-%m-%dT%H:%M:%S"
+    # iso_format = "%Y-%m-%dT%H:%M:%S"
```
Comment (yuvalbenshalom) [inline Integrations/IntSight/IntSight.py]: remove

File: Integrations/IntSight/IntSight.py [hunk 2]
```diff
@@ -310,7 +320,7 @@ def get_alert_activity():
     """
     alert_id = demisto.getArg('alert-id')
     r = req('GET', 'public/v1/data/alerts/activity-log/' + alert_id).json()
-    activities = []
+    # activities = []
```
Comment (yuvalbenshalom) [inline Integrations/IntSight/IntSight.py]: remove

File: Integrations/IntSight/IntSight.py [hunk 3]
```diff
@@ -635,7 +645,7 @@ def search_for_IOC():
     """
     Search for IOC by value
     """
-    value = demisto.getArg('value')
+    # value = demisto.getArg('value')
```
Comment (yuvalbenshalom) [inline Integrations/IntSight/IntSight.py]: remove

**Pull request merged**